test/callback.jl

```diff
@@ -53,16 +58,16 @@ end
   @test stats.iter == 8
 end
 
-@testset "Testing Solver Values" begin
-  f(x) = (x[1] - 1)^2 + 4 * (x[2] - x[1]^2)^2
-  nlp = ADNLPModel(f, [-1.2; 1.0])
-  function cb(nlp, solver, stats)
-    if stats.iter == 4
-      @test solver.σ > 0.0
-      stats.status = :user
-    end
-  end
-  stats = with_logger(NullLogger()) do
-    R2(nlp, callback = cb)
-  end
-end
+# @testset "Testing Solver Values" begin
+#   f(x) = (x[1] - 1)^2 + 4 * (x[2] - x[1]^2)^2
+#   nlp = ADNLPModel(f, [-1.2; 1.0])
+#   function cb(nlp, solver, stats)
+#     if stats.iter == 4
+#       @test solver.σ > 0.0
+#       stats.status = :user
+#     end
+#   end
+#   stats = with_logger(NullLogger()) do
+#     R2(nlp, callback = cb)
+#   end
```

Comment:
Why is this test commented out?

Reply:
The test is commented out for `solver` does not contain `σ` anymore, I'm not sure what to do with this test.

Reply:
I think that it is important we maintain existing features, so why not keep the `σ::T` in the solver?
Adding new solvers should add more tests not remove them :) .

Reply:
I did not keep `σ::T` variable, it has been replaced by alpha variable.
I can put the variable back in the solver structure (as alpha), and put the test back replacing sigma by alpha.

Reply:
Yes, let's go with alpha